package.json

```diff
@@ -20,25 +20,25 @@
         ]
     },
     "scripts": {
-        "_______ Hooks _______": "Yarn hooks.",
+        "_______ Hooks _______": "echo 'Yarn hooks.'",
```

Comment:
Please remove this "echo", this scripts are meant just as comments because normal comments are not possible in JSON and they will never be run. This is quite bad for readability of these comments.

Reply:
https://github.com/trezor/trezor-suite/pull/6005#discussion_r947977779
I understood its purpose but not having it wrapped in `echo` makes them invalid, they are not scripts. Seems like more people are using it like that https://stackoverflow.com/a/50878067/2428361

But sure, if you insist, I can revert those changes and remove shellchecking all these scripts.